build/app/public/js/base.js

```diff
@@ -12150,42 +12150,58 @@
   };
 
   // shared/js/ui/templates/shared/tracker-networks-text.js
-  function trackerNetworksText(requestDetails, protectionsEnabled) {
+  function trackerNetworksTitle(requestDetails, protectionsEnabled) {
     const state = requestDetails.state(protectionsEnabled);
     switch (state) {
       case states.protectionsOn_blocked:
       case states.protectionsOn_blocked_allowedTrackers:
       case states.protectionsOn_blocked_allowedNonTrackers:
       case states.protectionsOn_blocked_allowedTrackers_allowedNonTrackers: {
-        return {
-          title: ns.site("trackerNetworksDesc.title"),
-          icon: "blocked"
-        };
+        return ns.site("trackerNetworksDesc.title");
       }
       case states.protectionsOn_allowedTrackers_allowedNonTrackers:
       case states.protectionsOn_allowedTrackers:
       case states.protectionsOn_allowedFirstParty:
-      case states.protectionsOn_allowedFirstParty_allowedNonTrackers: {
-        return {
-          title: ns.site("trackerNetworksNotBlocked.title"),
-          icon: "info"
-        };
-      }
+      case states.protectionsOn_allowedFirstParty_allowedNonTrackers:
       case states.protectionsOff_allowedTrackers:
       case states.protectionsOff_allowedTrackers_allowedNonTrackers: {
-        return {
-          title: ns.site("trackerNetworksNotBlocked.title"),
-          icon: "warning"
-        };
+        return ns.site("trackerNetworksNotBlocked.title");
       }
       case states.protectionsOn:
       case states.protectionsOff:
       case states.protectionsOn_allowedNonTrackers:
       case states.protectionsOff_allowedNonTrackers: {
-        return {
-          title: ns.site("trackerNetworksNotFound.title"),
-          icon: "blocked"
-        };
+        return ns.site("trackerNetworksNotFound.title");
+      }
+      default:
+        return unreachable(state);
+    }
+  }
+  function trackerNetworksIcon(requestDetails, protectionsEnabled, phishingDetected) {
```

Comment:
@shakyShane I’ve decided to split Title and Icon logic as to not have the functions riddled with `if` statements. We end up with more switches, but the result of each function is less ambiguous